tensorflow/python/keras/BUILD

```diff
@@ -1622,7 +1615,6 @@ tf_py_test(
     ],
     shard_count = 8,
     tags = [
-        "no_cuda",
```

Comment:
same here. I saw weird errors in `keras:base_layer_test` on both `cuda` and `rocm` last week.

Reply:
same as above.

the only difference would be running locally vs running on CI servers.  The CI run should confirm it either way.